ghost/core/core/server/data/migrations/versions/5.45/2023-04-18-12-56-add-announcement-settings.js

```diff
@@ -0,0 +1,24 @@
+const {combineTransactionalMigrations, addSetting} = require('../../utils');
+
+module.exports = combineTransactionalMigrations(
+    addSetting({
+        key: 'announcement_content',
+        value: null,
+        type: 'string',
+        flags: 'PUBLIC',
```

Comment:
We have not added these 'PUBLIC' flags in a while. I think they are appropriate here as the field will be exposed through unauthenticated public API. But maybe we have changed thinking about the use of these that I have missed 🤔

Reply:
As discussed internally, we still keep using the `flags` field as we did not have a decision on removing it. Also made a fix to the migration utility making it possible to add these fields in migrations - https://github.com/TryGhost/Ghost/commit/7969399cdf99fe5d99bccf82084f2c97656bfe3c